firestore/query.go

```diff
@@ -569,8 +579,48 @@ func trunc32(i int) int32 {
 	return int32(i)
 }
 
+// Get returns an array of query's resulting documents.
+func (q Query) Get(ctx context.Context) ([]*DocumentSnapshot, error) {
+	limitedToLast := q.limitToLast
+
+	if q.limitToLast {
+		// Flip order statements before posting a request.
+		for i := range q.orders {
+			if q.orders[i].dir == Asc {
+				q.orders[i].dir = Desc
+			} else {
+				q.orders[i].dir = Asc
+			}
+		}
+		// Swap cursors.
+		q.startVals, q.endVals = q.endVals, q.startVals
+		q.startDoc, q.endDoc = q.endDoc, q.startDoc
+		q.startBefore, q.endBefore = q.endBefore, q.startBefore
+
+		q.limitToLast = false
```

Comment:
Why set this to `false` here? If this was not flipped could we just get rid of `limitedToLast`?